runtime/v2/runc/manager/manager_linux.go

```diff
@@ -270,8 +270,13 @@ func (manager) Stop(ctx context.Context, id string) (shim.StopStatus, error) {
 	if err := mount.UnmountAll(filepath.Join(path, "rootfs"), 0); err != nil {
 		log.G(ctx).WithError(err).Warn("failed to cleanup rootfs mount")
 	}
+	pid, err := runcC.ReadPidFile(filepath.Join(path, process.InitPidFile))
+	if err != nil {
+		log.G(ctx).WithError(err).Warn("failed to read init pid file")
+	}
```

Comment:
How about returning this error instead of logging? The function should return either a valid value or error.

Reply:
This function is used to cleanup the resource, which is handled in command-line.
It should not return error here.